reversion_compare/compare.py

```diff
@@ -23,6 +24,18 @@
 logger = logging.getLogger(__name__)
 
 
+@python_2_unicode_compatible
+class FieldVersionDoesNotExist(object):
+    """
+    Sentinel object to handle missing fields
+    """
+
+    def __str__(self):
+        return force_text(_("Field Didn't exist!"))
```

Comment:
I've added a commit for that!

cc https://github.com/jedie/django-reversion-compare/pull/54#discussion_r49726719 :-)